Freya Assignment jQuery/app.js

```diff
@@ -0,0 +1,36 @@
+$( document ).ready(function() {
+    alert('Click to browse our awesome page!');
+});
+
+$('.readmore').click(function() {
+	$('#show-this-on-click').slideDown();
+	$('.readmore').hide();
+	$('.hide').show();
```

Comment:
Both the readmore and learnmore parts uses the class ```.hide```, so when you click on the readmore, it also shows the learnmore part, which is a bug.